src/CBLReplicatorConfig.hh

```diff
@@ -268,9 +284,11 @@ namespace cbl_internal {
         }
 
 
-        alloc_slice      _pinnedServerCert, _trustedRootCerts;
-        CBLProxySettings _proxy;
-        alloc_slice      _proxyHostname, _proxyUsername, _proxyPassword;
+        CBLEndpoint*        _endpoint;
+        CBLAuthenticator*   _authenticator;
```

Comment:
What are these two for?  I don't think they are needed?

Reply:
We need them there so they can be deleted.

Reply:
Wait ... not sure what was in my mind. They are not used and we have kept both of them in endpoint and authenticator.